src/load.rs

```diff
@@ -59,20 +59,28 @@ use crate::Error;
 pub fn configurations_from_env() -> crate::Result<TomlValue> {
     let config_file = scan_directories_for_config_file()?;
 
-    let mut toml_value = match &config_file {
+    let toml_value = match &config_file {
         Some(file_path) => load_toml_from_path(file_path)?,
         None => TomlValue::Table(Default::default()),
     };
 
-    if let Ok(var) = env::var("BLOCKFROST_NETWORK_ADDRESS") {
-        toml_value["blockfrost-network-address"] = TomlValue::String(var);
-    }
+    Ok(toml_value.as_table().map_or(toml_value.clone(), |x| {
```

Comment:
Hi @b00kdev! I'll review the code. It seems that the essential fix is:
- using the correct env vars,
- using the correct config keys.

Do you think we could keep the original style of overriding the values? Has there been any problem with keeping the `mut` value and `[]` indexing? From what I've seen, `[]` should automatically work on `toml::Value`s that are tables.